### DISCUSSION TITLE
Custom emojis

### DISCUSSION THREAD

**Discussion by Its4Nik** [outline/outline#7104] [Ideas]
I would love to add custom emojis, like in discord

![image](https://github.com/outline/outline/assets/106100177/8b2ed00d-b8f7-4410-accd-1acc1b0eeda5)

**Comment by tommoor** [comment]
Seems inevitable :)

**Comment by Its4Nik** [comment]
> Seems inevitable :) 

Maybe even for the next release? 

(just asking cause that would be pretty sweet)

**Comment by cshaa** [comment]
This would be great, as there is no way to put inline images in the text afaik. Therefore if one wants to use custom icons in the text, they have to make do with emojis, which is quite limiting.

**Comment by inf0matics** [comment]
+1 on "this would be great". I use Icons as "tags" or Status indications. With custom emojis we can document our work better in outline

**Comment by VULGA01** [comment]
come on... i've been waiting for this for a year...

**Reply by Its4Nik** [reply]
> come on... i've been waiting for this for a year...

Dude chill it may seem easy but outline is a huge codebase. If you want it faster do it yourself and create a pr ffs

**Comment by JaredMerritt** [comment]
Would be great to have custom icons for collections and pages. Our biggest gripe coming from Notion!
